### PR TITLE
[paddle v2.0.0rc1: API fixs] assign/conv2d/conv2d_transpose/cast/ParamAttr

### PR DESCRIPTION
**Pull request by vslyu**
### PR types

Bug fixes
### PR changes

 APIs
### Describe

-  paddle.nn.functionl.assign 
-  paddle.static.nn.conv2d 
http://gzhxy-inf-bce36a39de.gzhxy.baidu.com:8121/documentation/docs/en/api/paddle/fluid/layers/conv2d_en.html
image
- paddle.static.nn.conv2d_transpose 
http://gzhxy-inf-bce36a39de.gzhxy.baidu.com:8121/documentation/docs/en/api/paddle/fluid/layers/conv2d_transpose_en.html
image
- paddle.cast 
image
- paddle.ParamAttr
http://gzhxy-inf-bce36a39de.gzhxy.baidu.com:8121/documentation/docs/en/api/paddle/fluid/param_attr/ParamAttr_en.html
image

### CODE REVIEW

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -3914,11 +3912,11 @@ def conv2d_transpose(input,
...
        .. code-block:: python
 
-          import paddle.fluid as fluid
           import paddle
           paddle.enable_static()
-          data = fluid.data(name='data', shape=[None, 3, 32, 32], dtype='float32')
-          conv2d_transpose = fluid.layers.conv2d_transpose(input=data, num_filters=2, filter_size=3)
+
+          data = paddle.static.data(name='data', shape=[None, 3, 32, 32], dtype='float32')
+          conv2d_transpose = paddle.static.nn.conv2d_transpose(input=data, num_filters=2, filter_size=3)
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/nn.py:3906]: 这里可以 print(conv2d_transpose.shape) # [shape]

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -1466,12 +1466,11 @@ def conv2d(input,
...
         .. code-block:: python
 
-          import paddle.fluid as fluid
           import paddle
           paddle.enable_static()
           
-          data = fluid.data(name='data', shape=[None, 3, 32, 32], dtype='float32')
-          conv2d = fluid.layers.conv2d(input=data, num_filters=2, filter_size=3, act="relu")
+          data = paddle.static.data(name='data', shape=[None, 3, 32, 32], dtype='float32')
+          conv2d = paddle.static.nn.conv2d(input=data, num_filters=2, filter_size=3, act="relu")
```
Comment (TCChenlong) [inline python/paddle/fluid/layers/nn.py:1484]: 这里可以 print(conv2d.shape) # [shape]

Comment (vslyu) [reply python/paddle/fluid/layers/nn.py:1484]: done,  "print(conv2d.shape) # [-1, 2, 30, 30]"

File: python/paddle/fluid/layers/nn.py (hunk 1 above)
Comment (vslyu) [reply python/paddle/fluid/layers/nn.py:3906]: done, "print(conv2d_transpose.shape) # [-1, 2, 34, 34]"

**Review by TCChenlong — DISMISSED** [review]
LGTM

**Review by zhiqiu — DISMISSED** [review]

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by zhiqiu — APPROVED** [review]
LGTM

**Pull request merged**